src/spas/libs/member-management/feature/src/lib/member-management/user-profile.ts

```diff
@@ -0,0 +1,11 @@
+export class User {
+  fullName: string;
+  emailAddress: string;
+  personalPhoneNumber: number;
+  homeAddress: string;
+  emergencyContactName: string;
+  emergencyContactPhone: number;
+  signUpDate: string;
+  congregationName: string;
+  groupNumber: number;
+}

```

Comment:
move this class into
` src/spas/libs/member-management/domain/src/lib/models`, and export the class via the member-management-api.ts or index.ts